skiko/src/jvmMain/kotlin/org/jetbrains/skiko/FPSCounter.kt

```diff
@@ -0,0 +1,44 @@
+package org.jetbrains.skiko
+
+import java.util.*
+import kotlin.math.roundToInt
+
+internal class FPSCounter(
+    private val count: Int,
+    private val probability: Double
+) {
+    private var i = 0
+    private val times = LinkedList<Double>()
+    private var t1 = System.nanoTime()
+
+    /**
+     * [value] 0.0 - min, 1.0 - max, 0.5 - median
+     */
+    private fun MutableList<Double>.quantile(value: Double) : Double {
+        sort()
```

Comment:
Mutating behavior here is rather confusing.

Reply:
Agree. Fixed